eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -493,7 +492,6 @@ class Peer(nodeParams: NodeParams, remoteNodeId: PublicKey, authenticator: Actor
   }
 
   onTransition {
-    case INSTANTIATING -> DISCONNECTED if nodeParams.autoReconnect && nextStateData.address_opt.isDefined => self ! Reconnect // we reconnect right away if we just started the peer
     case _ -> DISCONNECTED if nodeParams.autoReconnect => setTimer(RECONNECT_TIMER, Reconnect, 1 second, repeat = false)
```

Comment:
How about starting with `delay = 0 second` and let it be handled by  `nextReconnectionDelay` ? In this way even on mobile we're going to connect asap, but it might trigger herd effect on nodes with a lot of connections.